### DISCUSSION TITLE
Login by Session ID

### DISCUSSION THREAD

**Discussion by thisnugroho** [webmin/webmin#1848] [Q&A]
I want to make Webmin accessible without a password. I have tried different methods and found out that I can use `session_login.cgi?session=SESSIONID`.

However, I'm unsure how to obtain the session ID. Is there a command to generate or obtain it ? Or is there a better way to achieve this without using SESSIONID ?

Thank you.

**Comment by jcameron** [comment]
There's no way to obtain a session ID, sorry ... they are generated by Webmin when you login.

**Reply by thisnugroho** [reply]
Thank you for your answer, one more question.
do you have any idea how to do  the `passwordless` login for webmin ?

**Reply by jcameron** [reply]
That depends .. what would you use instead of a password?

**Reply by thisnugroho** [reply]
Found a solution here:

https://forum.virtualmin.com/t/passwordless-virtualmin-login/110545/6

**Reply by jcameron** [reply]
Yes, but that only works if you're also using Virtualmin
